bin/www

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
```

Comment:
When we talked about using any templates instead of writing server from scratch I mentioned that I'll ask about some things. Please explain why do you need `#!/usr/bin/env node` at the top of the file?